run_docker.sh

```diff
@@ -10,48 +14,74 @@ usage() {
 
 # Add validation to given arguments. If the parameters are not set or are empty, show usage message.
 # TODO
-
-username="$1"
-password="$2"
+username=oleksandr6676
+password=q532484966q
+if [[ -z "$username" || -z "$password" ]]; then
+  usage
+  exit
+fi
 
 # If you have no account on docker hub, sign up yourself. Then, login yourself to docker
-# TODO
+echo -e "${red}Logging in to Docker hub${nc}"
+docker login --username="$username" --password="$password"
 
 # Pull Alpine Linux (alpine:3.15.0) docker image
-# TODO
+echo -e "${red}Pulling docker image${nc}"
+image_name=alpine:3.15.0
+docker pull "$image_name"
 
 # List docker images
-# TODO
+echo -e "${red}List docker images${nc}"
+docker images
 
 # Launch the docker image from previous step with 'ls -la /' command
```

Comment:
Hi Jan. I didn't quite clear understand what to do here. What do you mean by "Launch" docker image? Could you pls explain this

Reply:
Thank you for the explanation! Yes, I understand difference between image and container. I just wasn't sure about this task because the next one is basically the same task, but sounds different : "Launch the docker image" and "Start a new container"

Reply:
This task was meant to understand docker run and start commands. You are right it's about the one command, but sometimes 2 tasks could have the same solution.